Library/Formula/zstd.rb

```diff
@@ -9,12 +9,15 @@ class Zstd < Formula
     "MIT", # lib/dictBuilder/divsufsort.c
   ]
 
+  # 4.0 doesn't support -Wvla -Wc++-compat
```

Comment:
GCC 4.0 is a separate `fails` with statement: 

```suggestion
  # 4.0 doesn't support -Wvla -Wc++-compat
  fails_with :gcc_4_0
```